src/containers/Application.js

```diff
@@ -176,7 +176,7 @@ it is accessible by the sidebar toggler button.
 	}
 
 
-	getApiURL(service) {
+	getServiceURL(service) {
```

Comment:
getServiceURL(service) {
		let services = this.Config.get('SERVICES');
		if (services == undefined) {
			return undefined;
		}

		service_path = services[service];

		// Handle if service is unknown => return undefined
		if (service_path == undefined || service_path == null) {
			console.log("Service path is undefined");
			return undefined;
		}
		// If service_path is complete URL, then return that URL
		if (service_path.toString().indexOf('http://') !== -1 || service_path.toString().indexOf('https://') !== -1) {
			return service_path;
		}

		let API_URL = this.Config.get('API_URL');
		return API_URL + "/" + service_path;
	}